front/src/common/ScenarioExplorer/ScenarioExplorer.tsx

```diff
@@ -7,11 +7,13 @@ import studyIcon from 'assets/pictures/views/study.svg';
 import { useTranslation } from 'react-i18next';
 import { MdTrain } from 'react-icons/md';
 import { useDispatch, useSelector } from 'react-redux';
-import { updateInfraID, updateTimetableID } from 'reducers/osrdconf';
 import { TrainScheduleSummary, osrdEditoastApi } from 'common/api/osrdEditoastApi';
 import { getDocument } from 'common/api/documentApi';
-import { getTimetableID } from 'reducers/osrdconf/selectors';
-import { useOsrdConfContext } from 'common/osrdConfContext';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
+import { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
import type { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
```

Reply:
Done